asistente_ladm_col/logic/ladm_col/config/reports/annex_17_report/pg/annex17_point_data_query.py

```diff
@@ -117,6 +189,6 @@ def get_annex17_point_data_query(names, schema, plot_id):
             ) AS f
         ) AS ff;""".format(**vars(names),  # Custom keys are searched in Table And Field Names object
```

Comment:
@gacarrillor I want share with that when we use **vars(names) in a format we must be careful that the key is not repeated.